src/cli/cluster/base_path_proxy.js

```diff
@@ -53,6 +53,28 @@ export default class BasePathProxy {
     server.route({
       method: '*',
       path: `${basePath}/{kbnPath*}`,
+      config: {
+        pre: [
+          (req, reply) => {
+            map(clusterManager.workers, (worker) => {
```

Comment:
Not yet, since we hadn't used them much we didn't want to pick styles upfront, but I'd be down to talk about standardization in another issue